source/docs/articles/organizations.md

```diff
@@ -5,22 +5,27 @@ category:
 - managing
 keywords: pantheon, pantheon for agencies, org, organization, org dashboard, change management
 ---
-Organizations on Pantheon bring together users, sites, custom upstreams, unified support, and Multidev to provide administrators with the tools needed to effectively manage a large number of websites. Several service levels are as types of organizations on our platform. Please see your organization dashboard or contact your sales representative to determine which type you are. Partners who signed up for "Pantheon for Agencies" via our self-service form are "Ally Partners." Ally Partners do not qualify for custom upstreams or free sites. Take a moment to read about all of the awesome we have available for your organization.
-
+Organizations on Pantheon bring together users, sites, custom upstreams,  support, and Multidev to provide administrators with the tools needed to effectively manage a large number of sites. Enterprise clients should contact their sales representatives for service levels and feature availability. Partner organizations can [sign up for free](https://pantheon.io/agency) and learn more about plans offered in our [Partner Program Guide](http://pantheon.io/sites/default/files/Partner_Program_Guide_2015.pdf).
 ## The Organization Dashboard
-Access the Organization Dashboard from the User Dashboard in the Organizations tab, between the site name and service level in an org-associated Site Dashboard, or in the **My Sites & Account** list by using the keyboard shortcut, "s". To perform the URL hack, type `dashboard.pantheon.io/org` and your browser will likely autocomplete the URL for your org dashboard. My fastest is a quick tap of the `s` key, followed by a move of the mouse to click on my org.
+Access the [Organization Dashboard](/docs/articles/organizations/pantheon-for-agencies/#tour-the-dashboard) in the following ways:
 
-## Multidev
+- From the User Dashboard, click the **Organizations** tab.
+- From the Site Dashboard, click the Organization's name next to your site's service level.
+- From any Dashboard location, hit the "s" keyboard shortcut to toggle a list of sites and organizations.
 
-All Partner organizations have Multidev available on every site. Click on **Multidev** in your Site Dashboard to get going. Sites have a maximum of ten Multidev environments and unlimited git branches, from which Multidev environments can be created on-demand. Learn how to [use Multidev](/source/docs/articles/sites/multidev).
+
+## Multidev
+All sites associated with any Partner organization will have [Multidev](/docs/articles/sites/multidev) enabled with up to 10 environments at their disposal.
```

Comment:
add caveat about multidev envs only being accessible for organization members, not site team member from outside org